src/kubo-deployment-tests/generate_manifest_test.go

```diff
@@ -36,39 +38,62 @@ var _ = Describe("Generate manifest", func() {
 	Context("successful manifest generation", func() {
 		kuboEnv := filepath.Join(testEnvironmentPath, "test_gcp")
 
-		DescribeTable("populated properties for CF-based deployment", func(line string) {
+		DescribeTable("populated properties for CF-based deployment", func(yPath, value string) {
 			cfEnv := filepath.Join(testEnvironmentPath, "test_vsphere_with_creds")
-			status, err := bash.Run("main", []string{cfEnv, "klingon", "director_uuid"})
+			status, _ := bash.Run("main", []string{cfEnv, "klingon", "director_uuid"})
 
+			var manifest map[string]interface{}
+			err := yaml.Unmarshal(stdout.Contents(), &manifest)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(status).To(Equal(0))
 
-			Expect(stdout).To(gbytes.Say(line))
+			template := boshtpl.NewTemplate([]byte(stdout.Contents()))
+			vars := boshtpl.StaticVariables{}
+			ops := patch.FindOp{Path: patch.MustNewPointerFromString(yPath)}
+
+			bytes, err := template.Evaluate(vars, ops, boshtpl.EvaluateOpts{ExpectAllKeys: false})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(string(bytes[:len(bytes)-1])).To(Equal(value))
 		},
-			Entry("deployment name", "\nname: klingon\n"),
-			Entry("network name", "\n  networks:\n  - name: network-name\n"),
-			Entry("kubernetes external port", "\n      external_kubo_port: 101928\n"),
-			Entry("CF API URL", "\n        api_url: cf.api.url\n"),
-			Entry("CF UAA URL", "\n        uaa_url: cf.uaa.url\n"),
-			Entry("CF Client ID", "\n        uaa_client_id: cf.client.id\n"),
-			Entry("CF Client Secret", "\n        uaa_client_secret: cf.client.secret\n"),
-			Entry("Auto-generated kubelet password", "\n      kubelet-password: \\(\\(kubelet-password\\)\\)\n"),
-			Entry("Auto-generated admin password", "\n      admin-password: \\(\\(kubo-admin-password\\)\\)\n"),
+			Entry("deployment name", "/name", "klingon"),
+			Entry("network name", "/instance_groups/name=etcd/networks/0/name", "network-name"),
+			Entry("Master Kubeconfig Kubernetes API URL", "/instance_groups/name=master/jobs/name=kubeconfig/properties/kubernetes-api-url", "https://a.router.name:101928"),
```

Comment:
Hey @srm09 @tvs - it looks like this test is failing because kubernetes-api-url has been removed when we introduced bosh dns